wrapper/src/template.sh

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
 
 {
+    BATECT_QUIET_DOWNLOAD=${BATECT_QUIET_DOWNLOAD:-false}
```

Comment:
Could you please move this down with the other variables? The `set -euo pipefail` line should go first.

Reply:
Oh nice, I had it before to do that variable check like before, but this doesn't trip up the pipefail situation